viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -48,6 +51,44 @@
       projectCollection.getProjects();
     }
 
+    var filterByUser = function () {
+      var input = $('#userNameBox').val();
+      var rows = $('#project-list').find('tr');
+      if (input === "") {
+        rows.show();
+        return;
+      }
+      rows.hide();
+      var filteredRowIds = _.uniq(rows.filter(function () {
+        var $t = $(this);
+        return $t.find('#innerCreatedBy').find("label").text().toLowerCase().indexOf(input.toLowerCase()) !== -1;
+      }).map(function () {
+        var row = $(this);
+        return row.attr('id');
+      }));
+      rows.filter(function () {
+        var $t = $(this);
```

Comment:
Why this var is needed?

Reply:
This one is not needed, changing.

Reply:
This one is needed because the context changes when you are inside the _.contains